odoo/tests/common.py

```diff
@@ -702,15 +702,26 @@ def executable(self):
                     return bin_
 
         elif system == 'Windows':
-            # TODO: handle windows platform: https://stackoverflow.com/a/40674915
-            pass
+            bins = [
+                '%ProgramFiles%\Google\Chrome\Application\chrome.exe',
+                '%ProgramFiles(x86)%\Google\Chrome\Application\chrome.exe',
+                '%LocalAppData%\Google\Chrome\Application\chrome.exe',
+            ]
+            for bin_ in bins:
+                bin_ = os.path.expandvars(bin_)
+                if os.path.exists(bin_):
+                    return bin_
 
         raise unittest.SkipTest("Chrome executable not found")
 
     def _spawn_chrome(self, cmd):
-        if os.name != 'posix':
+        if os.name == 'nt':
+            proc = subprocess.Popen(cmd)
+            pid = proc.pid
+        elif os.name == 'posix':
+            pid = os.fork()
+        else:
```

Comment:
```suggestion
        
       if os.name == 'nt';
            proc = subprocess.Popen(cmd)
            return proc.pid
       if os.name != 'posix':
           return
       pid = os.fork()
```

- This will minimise the diff
- This makes more sense I think since the check `if pid != 0:` is really specific to the fork and doesn't really makes sence after the subprocess.Popen. It is more correct to return the pid IMHO


Actually since the fork is only relevant to manage memory limit in posix, we could go on something like that
```suggestion
       if os.name != 'posix':
            proc = subprocess.Popen(cmd)
            return proc.pid
       # manage memory limit for posix
       pid = os.fork()
```

Just a suggestion, would like to have @xmo-odoo  and @d-fence opinions on the pr anyway.

Reply:
@Xavier-Do Thank you for review.
I consider your first suggestion.
According to documentation `os.name` can be `nt`, `posix` or `java`.
So variant to simply check `os.name != 'posix'` for subprocess.Popen is too risky for me.
I left strict check `os.name == 'nt'` cause I didn't test this PR on java environment and even don't know is there option to run Odoo in java environment.

Can you ping @odoo/rd-security please? This user is not visible for me.

Reply:
I pinged @xmo-odoo , he's part of the security and is the one for tests/common.py I think ;)

Reply:
I think we don't really care about java but you have a point.  

small note: 
Previously the Popen call was `subprocess.Popen(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)`. looks like we want to silent chrome to avoid log spams.

Reply:
> I left strict check os.name == 'nt' cause I didn't test this PR on java environment and even don't know is there option to run Odoo in java environment.

There is not. AFAIK "java" is for jython, on which odoo has never run, and on which it's unlikely to ever do so (as it's pretty poorly if not outright dead). We can just ignore it entirely, it does not matter.

Reply:
@Xavier-Do sorry, I confused
I was forced to rollback your suggestion because of possible forwardporting
Starting from 14.0 the `if pid != 0:` branch do more than just return (it determines devtools port)
So we can't just return pid for `nt` case and should pass to the rest of method.
For 13.0 it is really not necessary but in case of forwardporting current version of code will be universal.